src/main/drivers/timer_stm32f4xx.c

```diff
@@ -41,7 +41,7 @@ const timerDef_t timerDefinitions[HARDWARE_TIMER_DEFINITION_COUNT] = {
     { .TIMx = TIM5,  .rcc = RCC_APB1(TIM5),  .inputIrq = TIM5_IRQn},
     { .TIMx = TIM6,  .rcc = RCC_APB1(TIM6),  .inputIrq = 0},
     { .TIMx = TIM7,  .rcc = RCC_APB1(TIM7),  .inputIrq = 0},
-#if !defined(STM32F411xE) && !defined(STM32F446xx)
```

Comment:
Not entirely sure about this one (and I don't even have a STM32F446 to test with), but according to the datasheet the STM32F446 does have timer 8. @jflyper do you know more?

Reply:
F446 does have TIM8, and targets `NUCLEOF446RE` and `SPEKTRUMF400` both use TIM8.

I wonder how these targets could have worked with this conditional ...

Reply:
I doubt anybody ever used `NUCLEOF446RE` in a craft. `SPEKTRUMF400` was abandoned hardware at the time we added support for it, so potentially there aren't that many users of it out there.

Reply:
Ok, looks like TIM8 was left out on STM32F446 because it does not have an IRQ number for capture compare on TIM8, which is odd considering the datasheet shows that TIM8 is available on pins. Not sure what IRQ number is used for this, setting to 0 for now.
@jflyper: Do you have a board with F446 that you can use to test?